index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav class="navbar">
+        <div>
+            <h3 class="my-logo">My logo</h3>
+        </div>
+
+       <div class="nav-right">
+           <img src="images/Enabled.png" alt="normal button">
+       </div>
+    </nav>
```

Comment:
- [ ] Kindly wrap this section with a header tag, only the hamburger menu on the right should have a nav tag.
- [ ] Kindly ensure your logo has a tag, it should be clickable.
- [ ] Kindly add the background images to your design to match the design given

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/170294826-3ada6d2e-b87f-48f1-9f4d-4847d376bea9.png)|  ![image](https://user-images.githubusercontent.com/61408860/170294897-0bf014f3-d322-4848-9fe6-6f7c15be6cd9.png)